pkg/stub/handler.go

```diff
@@ -862,6 +960,73 @@ func (h *Handler) processError(opcfg *v1alpha1.SamplesResource, ctype v1alpha1.S
 	return err
 }
 
+func (h *Handler) imageStream(imagestream, is *imagev1.ImageStream, opcfg *v1alpha1.SamplesResource) error {
```

Comment:
this method name is not very enlightening....  also the two imagestream parameters named "imagestream" and "is" leave me unclear on the difference between the two.

Reply:
method names will be prefaced with `upsert`

parameters names will be suffixed by `InOperatorImage` and `InCluster`